docs/content/v0.20/README.md

```diff
@@ -48,8 +47,18 @@ See [Packaging and Delivery](packaging.md)
 
 ## Usage examples, and testing
 
-See [Examples](examples.md) and [integration testing](contributor.md#integration-testing) for _this_ release, after reviewing [above](#latest-stable-release).
+[Examples](examples.md) shows how to use a release of KubeStellar.
```

Comment:
Maybe:
shows how to install the KubeStellar environment and describe the step by step instructions for few usage examples. 

Why is it important to explicitly mention "the release"  ?  All the documentation (except the developer section that show how to build and test the code) is expected to use releases

Reply:
I also added a little elaboration here, like the suggested additional information.